src/utils/event-manager/wheel-input.js

```diff
@@ -64,6 +64,7 @@ export default class WheelInput {
     if (!this.options.enable) {
       return;
     }
+    event.preventDefault();
```

Comment:
Yeah, [here](https://github.com/uber/react-map-gl/pull/283).
> Fix bug where page scroll is blocked even with scrollZoom: false

Oh, is the idea that instead of blocking it, we're now conditionally adding the handler only when `scrollZoom` is `true`?